inst/odin/compare_basic.cpp

```diff
@@ -0,0 +1,30 @@
+#include <dust/densities.hpp>
```

Comment:
Have we been using headers in included snippets? i.e. does the vaccine code use dust header files too?
Might be difficult to remember/know which one to pick for the non-dust developer, so we could either just make sure this is documented, or we could presumably add this header in to dust.hpp like we do with the distributions.

Reply:
We've managed to avoid the snippets approach to this point, but yeah, this is something that we should be able to handle ourselves. The tidiest way would be for odin.dust or dust to include this for us I think https://github.com/mrc-ide/odin.dust/issues/58

Reply:
These headers now come directly from dust so are removed here; I think this is a good call